packages/store-api/src/typeDefs/index.ts

```diff
@@ -0,0 +1,39 @@
+import { print } from 'graphql'
+
+import AggregateOffer from './aggregateOffer.graphql'
+import AggregateRating from './aggregateRating.graphql'
+import Author from './author.graphql'
+import Brand from './brand.graphql'
+import Breadcrumb from './breadcrumb.graphql'
+import Collection from './collection.graphql'
+import Facet from './facet.graphql'
+import Image from './image.graphql'
+import Offer from './offer.graphql'
+import Organization from './organization.graphql'
+import PageInfo from './pageInfo.graphql'
+import Product from './product.graphql'
+import ProductGroup from './productGroup.graphql'
+import Query from './query.graphql'
+import Review from './review.graphql'
+import Seo from './seo.graphql'
+
+export const typeDefs = [
```

Comment:
How about calling this `graphqlTypes` or something like that? I think we should avoid abbreviations as much as possible. Could also be `typeDefinitions` hahahaha no problem

Reply:
I think we must keep `typeDefs` since is a common name for that.

> In src/schema.js, import gql from apollo-server and create a variable called typeDefs for your schema:

https://www.apollographql.com/docs/tutorial/schema/

Reply:
This is the standard way for calling graphql type definitions. Apollo and graphql-tools and graphql-js use this standard

Reply:
Awesome then! 😄